src/mach/mod.rs

```diff
@@ -464,3 +491,59 @@ impl<'a> Mach<'a> {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::{Mach, MultiArchEntry};
+
+    #[test]
+    fn parse_multi_arch_of_macho_binaries() {
+        // Create via:
+        // gcc -arch arm64 -o /tmp/hello_world_arm hello_world.c
```

Comment:
clang is likely more accurate here on macos systems?

Reply:
Fair call, `gcc` is really `clang` on my laptop apparently:

```
$ gcc --version
Apple clang version 13.1.6 (clang-1316.0.21.2.5)
Target: arm64-apple-darwin21.1.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```